### ISSUE TITLE
example: lapd-drift does not give any result

### ISSUE THREAD

**Issue by bellsis83** [boutproject/BOUT-dev#1945]
Hi All,

I am still trying to run the example to understand BOUT++. While other examples give reasonable results, the lapd-drift does not give any result- I mean if I plot or print the values 'ni' or others, it gives a zero matrix with 401x52x32x32. 
I know that it is a too specific question but I would appreciate if you give me any comment.

Thanks!

**Comment by bendudson** [comment]
Hi Sorry about that. I suspect this model isn't used much; it was using a way of getting inputs which hasn't been supported for many years:
```
xs_opt = 3
ys_opt = 3
zs_opt = 3
xs_mode = 1 # Radial mode number
ys_mode = 1 # Parallel mode number
zs_mode = 1 # asimuthal mode number
```
which meant that the starting perturbations were all set to zero. These inputs were long ago replaced by expressions e.g.
```
function = mixmode(x)*mixmode(y)*mixmode(z)
```
which does essentially the same thing.
With the fixes in PR #1950 the simulations run again with non-zero output.

**Comment by ZedThree** [comment]
Fixed in #1950

**Comment by bellsis83** [comment]
@bendudson 
I appreciate your assist!
